frontend/package.json

```diff
@@ -26,7 +26,7 @@
     "react-redux": "7.1.3",
     "react-redux-loading-bar": "4.6.0",
     "react-router-dom": "5.1.2",
-    "react-scripts": "3.3.0",
+    "react-scripts": "3.4.0",
```

Comment:
The latest `react-scripts` is 3.4.1. If we are updating this it should be to the latest version I think?

Reply:
Nope no specific reason, thank you for catching that!